nativescript-angular/router/router.module.ts

```diff
@@ -15,9 +15,8 @@ import { NSEmptyOutletComponent } from "./ns-empty-outlet.component";
 export { PageRoute } from "./page-router-outlet";
 export { RouterExtensions } from "./router-extensions";
 export { NSModuleFactoryLoader } from "./ns-module-factory-loader";
-export { NSEmptyOutletComponent } from "./ns-empty-outlet.component";
 
-export type LocationState = LocationState;
```

Comment:
Is this a breaking change, from where do we export this?

Reply:
it is not, this change will go in the new scoped package only, the compatibility one will have the type exported from this path